language/move-lang/tests/move_check/locals/assign_resource.exp

```diff
@@ -15,6 +15,12 @@ error:
    ·
  5 │         let r = R{};
    │             - The local contains a value due to this assignment. The value does not have the 'drop' ability and must be used before you assign to this local again
+   ·
+ 5 │         let r = R{};
+   │                 --- The type '0x8675309::M::R' does not have the ability 'drop'
```

Comment:
I noticed that the `{}` is also underlined in the error message, and I don't think this should be intentional?

Reply:
It's the location of where the type was created. So I think you want the entire pack?

Reply:
I was thinking maybe just underling the `R` would do? I am more curious what will happen if some `R { ... }` spans multiple lines. But I might be wrong.

Reply:
It will do something abit wonky, sure. 

We don't currently have the loc for the name + any type arguments, so for `Cup<T> {}` you would only underline the `Cup`, which is definitely not what you want for that case. So we currently underline everything, which feels alright as it is the expression where the type came from